microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonDeployableContainer.java

```diff
@@ -422,6 +455,22 @@ public void undeploy(Descriptor descriptor) {
         // No-Op
     }
 
+    /**
+     * Injects the base metric registry and cleans up all metrics in preparation to run another
+     * Arquillian test in the same VM. Without this cleanup, metrics added by a previous test
+     * would be available and may cause failures.
+     */
+    private void cleanupBaseMetrics() {
```

Comment:
This change seems to be unrelated to LRA. Is it really needed?

Reply:
I have also backported chunk of changes to arqillian deployer to avoid h2 to be generated outside of the target folder, metrics cleanup invocation was part of those changes with aborted deployment cleanup and etc, didn't want to backport just a part of it. 
These changes greatly improve test isolation which is mostly visible only when some tck test fails - only test related to particular change fail, not all the other broken by polluted deployment.